pkg/kubelet/dockershim/docker_sandbox.go

```diff
@@ -216,14 +216,34 @@ func (ds *dockerService) StopPodSandbox(podSandboxID string) error {
 // sandbox, they should be forcibly removed.
 func (ds *dockerService) RemovePodSandbox(podSandboxID string) error {
 	var errs []error
+	opts := dockertypes.ContainerListOptions{All: true}
+
+	opts.Filter = dockerfilters.NewArgs()
+	f := newDockerFilter(&opts.Filter)
+	f.AddLabel(sandboxIDLabelKey, podSandboxID)
+
+	containers, err := ds.client.ListContainers(opts)
+	if err != nil {
+		errs = append(errs, err)
+	}
+
+	// Remove all containers in the sandbox.
+	for i := range containers {
+		if err := ds.RemoveContainer(containers[i].ID); err != nil && !dockertools.IsContainerNotFoundError(err) {
```

Comment:
@Random-Liu Use `ds.RemoveContainer()` here instead of `ds.client.RemoveContainer()` to use contianer remove logic.